delete_this_example.py

```diff
@@ -0,0 +1,27 @@
+import sublime_plugin
+from LSP.event_loop import run_future
+from .view_async import open_view, save_view
+from pathlib import Path
+
+class LspExampleAsyncCommand(sublime_plugin.TextCommand):
```

Comment:
if you want to try this command, here is a keybinding `{"keys": ["primary+i"], "command": "lsp_example_async" }` or `view.run_command('lsp_example_async')` if you want to run it from the ST console.